.github/workflows/pr-test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        go: ["1.17", "1.18", "1.19"]
+        go: ["1.18", "1.19"]
```

Comment:
Add "1.20"

Reply:
Done

For future reference, where did you read the 1.20 support?  I used this link for reference to know 1.17 was gone. I did not see 1.20.
https://github.com/open-telemetry/opentelemetry-go

Reply:
In the compatibility it says latest version - 2. https://github.com/open-telemetry/opentelemetry-go#compatibility

Reply:
For posterity, otel hasn't updated their compatibiity chart nor their CI yet. It still only runs golang 1.18/1.19.  However, golang 1.20 was released Feb 1st and otel points to https://go.dev/doc/devel/release#policy to say they support the last 2 golang versions. 

It does not hurt to add this into our CI right now. 

Done.